### ISSUE TITLE
Renamed characters lose notes

### ISSUE THREAD

**Issue by LurkerHub** [fcwill/fchat-pidgin#194]
When a bookmarked character is renamed, the notes attached to it disappear.

**Comment by vobe** [comment]
I'm not sure what notes you are talking about, could you clarify the use case? Screenshots might help

**Comment by LurkerHub** [comment]
![image](https://user-images.githubusercontent.com/20542915/184988124-2e5cab78-c8e1-436c-8f6b-7a4394c8e20a.png)

```XML
<contact>
	<buddy account='User:Character' proto='prpl-flist'>
		<name>Character about to be renamed</name>
		<setting name='notes' type='string'>These are the notes that are lost</setting>
	</buddy>
</contact>
```

**Comment by vobe** [comment]
Right, I wasn't aware there was a plugin for that.

I tried to right-click>alias on a character with notes, and the notes remained. Moving  the character to a different group also keeps notes. What do you mean by renamed?

If you mean a server-side rename initiated by your bookmarked buddy, I'm afraid there is not much we can do. The protocol considers them as a different user

**Comment by alexolog** [comment]
> If you mean a server-side rename initiated by your bookmarked buddy, I'm afraid there is not much we can do. The protocol considers them as a different user

Yes, that's what I meant.  It would have been nice to save the character id and update the old contact instead of deleting and adding

**Comment by LurkerHub** [comment]
Server-side rename.

**Comment by LurkerHub** [comment]
So a server-side rename is a combination of delete + make new, as far as the protocol is concerned?
I do get an email notification that a bookmarked character was renamed, so there must be something on the server side that knows that.

I guess that it's a part of a bigger issue - the removal of deleted characters.
If a character I bookmarked/friended is deleted, how can I keep the "buddy" in order to have access to old chats?
Moving it to a different group (deleted?) would be nice
